controllers/machinepool_controller.go

```diff
@@ -76,16 +76,6 @@ func (r *MachinePoolReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, nil
 	}
 
-	// We just switched the type that we are adding the finalizer to.
```

Comment:
Let's put this in the changelog and commit message (or separate in another PR)

Reply:
Done